src/dotnet/Audio.Service/AudioHubBackend.cs

```diff
@@ -0,0 +1,54 @@
+using ActualChat.Transcription;
+using Microsoft.AspNetCore.SignalR;
+
+namespace ActualChat.Audio;
+
+public class AudioHubBackend : Hub
+{
+    private AudioStreamServer AudioStreamServer { get; }
+    private TranscriptStreamServer TranscriptStreamServer { get; }
+
+    public AudioHubBackend(AudioStreamServer audioStreamServer, TranscriptStreamServer transcriptStreamServer)
```

Comment:
why class deps? in AudioHub you reference IAudioStreamServer, but here - AudioStreamServer

Reply:
Because contract interface has been implemented by Proxy - and in this case we would like to deal with actual implementation only